de.js

```diff
@@ -0,0 +1,334 @@
+    delete_wallet: `Wallet löschen`,
+    sign_in_to_the_dogeverse: `Melden Sie sich beim Dogeverse an`,
+    dogeverse_intro_description: `Fordern Sie Ihren DogeTag an, finden Sie Freunde, erhalten Sie Zahlungen mit Ihrem Link und interagieren Sie mit Inhalten im Dogeverse.`,
+    sign_in: `anmelden`,
+    sign_in_description: `Ihre Telefonnummer wird nur zur Anmeldung verwendet und nicht veröffentlicht.`,
+    create_account_description: `Treten Sie dem Dogeverse bei und verbinden Sie sich mit einem riesigen Netzwerk von Shibes.`,
+    send_verification: `Bestätigung senden`,
+    verify_your_phone_number: `Bestätige deine Telefonnummer`,
+    enter_the_code_sent_to: `Geben Sie den Code ein, der gesendet wurde an`,
+    resend_code: `Code erneut senden`,
+    wrong_number: `Falsche Nummer`,
+    confirm_code: `Code bestätigen`,
+    youve_entered_the_verse: `Du bist in den Vers eingetreten`,
+    youre_now_ready_to_explore_the_dogeverse: `Jetzt sind Sie bereit, das Dogeverse zu erkunden!`,
+    lets_go: `Lass uns gehen!`,
+    edit_profile: `Profil bearbeiten`,
+    social_connect: `Soziale Verbindung`,
+    profile_privacy: `Profil Datenschutz`,
+    profile_is_public: `Profil ist öffentlich`,
+    show_wallet_address_and_tip_button: `Wallet-Adresse und Trinkgeld-Button anzeigen`,
+    preferences: `Einstellungen`,
+    confirm_x_tips: `Bestätigen Sie 𝕏 Tipps`,
+    tips_received: `Erhaltene Trinkgelder`,
+    new_followers: `Neue Follower`,
+    new_memes: `Neue Memes`,
+    about_mydoge: `Über MyDoge`,
+    terms_of_use: `Nutzungsbedingungen`,
+    privacy_policy: `Datenschutzrichtlinie`,
+    share_mydoge: `MyDoge teilen`,
+    visit_our_website: `Besuche unsere Webseite`,
+    settings_and_support: `Einstellungen und Support`,
+    connect_twitch: `Twitch verbinden`,
+    connect_x: `Verbinden 𝕏`,
+    create_a_post: `Einen Beitrag erstellen`,
+    following: `Gefolgt`,
+    followers: `Anhänger`,
+    recent_activity: `Letzte Aktivität`,
+    history: `Geschichte`,
+    price: `Preis`,
+    usd: `US Dollar`,
+    txs: `TXs`,
+    hr: `Std`,
+    send: `Schicken`,
+    receive: `Erhalten`,
+    buy: `Kaufen`,
+    scan: `Scan`,
+    from: `Aus`,
+    to: `Zu`,
+    note: `Notiz`,
+    timestamp: `Zeitstempel`,
+    confirmations: `Bestätigungen`,
+    tx_hash: `TX-Hash`,
+    tx_hash_copied: `Transaktions-Hash kopiert`,
+    note_copied: `Notiz kopiert`,
+    address_copied: `Adresse kopiert`,
+    wallet: `Geldbörse`,
+    dogeverse: `Dogeverse`,
+    shop: `Geschäft`,
+    receive_doge: `Doge erhalten`,
+    send_doge: `Senden Sie Doge`,
+    confirm_transaction: `Transaktion bestätigen`,
+    scan_qr_code: `QR-Code scannen`,
+    new_post: `Neuer Beitrag`,
+    send_doge_alert_box_message: `Sie können DOGE an Ihre Freunde senden, eine Wallet-Adresse oder einen MyDoge-Benutzernamen eingeben`,
+    wallet_address_or_username: `Wallet-Adresse oder Benutzername`,
```

Comment:
same here, maybe change in English / other languages too :)
`    wallet_address_or_username: 'Wallet-Adresse oder DogeTag',`